File: android/app/src/debug/java/com/laneshadow/sandbox/stories/templates/TemplateStories.kt
package com.laneshadow.sandbox.stories.templates

import com.nativesandbox.model.Story

/**
 * Template tier stories for the LaneShadow sandbox.
 *
 * Template stories include:
 * - Screen layouts
 * - Full component compositions
 * - Page templates
 *
 * See UC-SCR-01 through UC-SCR-06 for detailed screen stories.
 */
object TemplateStories {
    val all: List<Story> = IdleScreenStory.all + PlanningScreenStory.all + PlaceholderTemplateStories.all
}

File: ios/LaneShadow/Sandbox/Stories/TemplateStories.swift
import LaneShadowTheme
import NativeSandbox
import SwiftUI

/// Template stories — Navigator screen templates.
@MainActor
enum TemplateStories {
    static let all: [Story] = {
        var stories: [Story] = []

        // Sprint 6: Navigator screen templates
        stories.append(contentsOf: IdleScreenStory.all)

        return stories
    }()
}
